src/content/blog/astro-330.mdx

```diff
@@ -0,0 +1,120 @@
+---
+title: "Astro 3.3: <Picture /> component"
+description: "Astro 3.3 is out with a brand new, experimental `<Picture />` component, improved compatibility for syntax highlighting, and package provenance."
+publishDate: "October 12, 2023"
+authors:
+  - nate
+  - erika
+  - bjorn
+  - ema
+coverImage: "/src/content/blog/_images/astro-330/blog-hero-3_3-release.webp"
+socialImage: "/src/content/blog/_images/astro-330/og-image-3_3-release.webp"
+lang: "en"
+---
+
+Astro 3.3 is out with a brand new, experimental `<Picture />` component, improved compatibility for syntax highlighting, package provenance, and a handful of other quality of life improvements.
+
+- **[New `<Picture />` component](#picture-component)**
+- **[Syntax Highlighting improvements](#syntax-highlighting)**
+- **[Package Provenance](#package-provenance)**
+- **[Quality of Life Improvements](#quality-of-life-improvements)**
+
+```
```

Comment:
The 3.2 post has a small introductory paragraph before the code snippet for a little extra context. Maybe we could add that here too?

````suggestion
To take advantage of the latest features, make sure you’re running the latest version of Astro. You can upgrade to Astro 3.3 by running the upgrade command for your package manager of choice:

```sh
````